src/main/resources/application.yml

```diff
@@ -103,5 +103,5 @@ aat:
   test-user:
     username: "${AAT_TEST_USERNAME:}"
     password: "${AAT_TEST_PASSWORD:}"
-  test-instance:
-    uri: "${TEST_URL:}"
+  test-instance-uri: "${TEST_URL:}"
+  test-user-email-pattern: "${AAT_TEST_USER_EMAIL_PATTERN:}"
```

Comment:
why not just let spring auto convert to an env variable, same above?

Reply:
I could change it but personally I'm not too fond of implicit configuration, makes the code harder to understand for people who are not familiar with this Spring magic. Why do you prefer the implicit way?

Reply:
I am in favour of convention that allows overriding application configuration with the env variable of the same name. On one hand it is well known part of the Spring framework and other Reform teams already using it. On the other hand it does not make us to invent two names for the same thing. Finally IMO it forces us to think about the structure of the configuration more and enforces better names.

Reply:
Ok, will update them 👍